packages/dd-trace/src/appsec/remote_config/index.js

```diff
@@ -23,9 +24,32 @@ function enable (config, appsec) {
       rc.updateCapabilities(RemoteConfigCapabilities.ASM_ACTIVATION, true)
     }
 
-    rc.setProductHandler('ASM_FEATURES', (action, rcConfig) => {
+    rc.updateCapabilities(RemoteConfigCapabilities.ASM_AUTO_USER_INSTRUM_MODE, true)
+
+    let autoUserInstrumModeId
+
+    rc.setProductHandler('ASM_FEATURES', (action, rcConfig, configId) => {
       if (!rcConfig) return
 
+      // this is put before other handlers because it can reject the config
+      if (typeof rcConfig.auto_user_instrum?.mode === 'string') {
+        if (action === 'apply' || action === 'modify') {
+          // check if there is already a config applied with this field
+          if (autoUserInstrumModeId && configId !== autoUserInstrumModeId) {
+            // eslint-disable-next-line no-throw-literal
```

Comment:
should we add an error? something like:
`log.error('[RC] Discarding config. Multiple auto_user_instrum.mode received')`

Reply:
Fixed in [98cae84](https://github.com/DataDog/dd-trace-js/pull/4674/commits/98cae84a93d3efb150b93273d207bd15e619572d)